internal/auth0/log_mock.go

```diff
@@ -5,74 +5,75 @@
 package auth0
 
 import (
+	reflect "reflect"
+
 	gomock "github.com/golang/mock/gomock"
 	management "gopkg.in/auth0.v5/management"
-	reflect "reflect"
 )
 
-// MockLogAPI is a mock of LogAPI interface
```

Comment:
there are a lot of this mock file changes, I wonder if you used a new version. Do we have the tool vendored?

Reply:
Yes, I used the latest version of mockgen. It's not vendored:
<img width="543" alt="Screen Shot 2021-06-21 at 20 45 38" src="https://user-images.githubusercontent.com/5055789/122847867-da2eb980-d2de-11eb-9faa-2da2dc9df313.png">
I removed the newly generated mocks and reverted the exiting ones.
Good catch!